cmd/cbuild/root.go

```diff
@@ -20,8 +20,10 @@ import (
 
 var version string
 
+const versionNotice = " (C) 2022 Arm Ltd. and Contributors"
+
 func printVersion(file io.Writer) {
-	fmt.Fprintf(file, "cbuild version %v\n", version)
+	fmt.Fprintf(file, "cbuild %v %v\n", version, versionNotice)
```

Comment:
In the formatted string, there will be 2 spaces between the version and versionNotice.